pieforproviders/pieforproviders
# [PIE-1145] Adds remaining units columns to dashboard

Adds remaining units columns, and adds approval dates to endpoint return json

## 💅🏼 What issue does this fix?



Closes #1145 - this adds the approval start/end dates to the dashboard API endpoint, new columns to show information about the approval period & remaining units, and Spanish translations

Closes #1658 - definitions

## 🍂 Before You Submit

* [X] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [X] Does your PR contain any required translations?

## 🧳 Steps to test
- Visit the dashboard with `nebraska@test.com` and `admin@test.com` to see the Nebraska dashboard
- Scroll all the way to the right
- There should be three new columns for Authorized Period, Total Hours Remaining and Total Days Remaining, as well as a new column header for the Total Authorized Period
- The columns should be sortable, by effective date, hours remaining, and days remaining
- This should work in English and Spanish
- This should work in *both* accounts (admin & nebraska)
